client/src/components/new-request/analyze-results.vue

```diff
@@ -270,13 +301,115 @@ export default class AnalyzeResults extends Vue {
     }
     return []
   }
+  get designations () {
+    if (this.issue && this.issue.designations) {
+      return this.issue.designations
+    }
+    return null
+  }
   get designationIsFixed () {
```

Comment:
This getter was previously living in grey-boxes.vue and had a watcher to update the store when its value changed because both this component (parent comp) and grey-boxes.vue (child component) relied on it.  Obviously makes more sense to have it exist in parent and pass to child as a Prop.